science/ngspice/Portfile

```
@@ -19,38 +18,48 @@ long_description \
 homepage        http://ngspice.sourceforge.net/
 master_sites    sourceforge:project/ngspice/ng-spice-rework/${version}
 
-checksums       rmd160  094d76e789b65d10e7d80f2f2f90397ed0ec13c9 \
-                sha256  4f818287efba245341046635b757ae81f879549b326a4316b5f6e697aa517f8c \
-                size    8450071
+# generated via openssl dgst -rmd160 ngspice-43.tar.gz 
+
+checksums       rmd160  577a26e18e70fdf1fceb87ae3c6a783c3cf00d8e \
+                sha256  14dd6a6f08531f2051c13ae63790a45708bd43f3e77886a6a84898c297b13699 \
+                size    10279606
 
 set docdir      ${prefix}/share/doc/${name}
 
 if {${name} eq ${subport}} {
 
     # freetype2 headers are not found by default
-    # we could fix the header path in configure.ac, but then need autorecconf, etc
-    configure.cppflags-prepend -I${prefix}/include/freetype2
-
-    # TASK_BASIC_INFO_COUNT and friends were renamed in 10.8
-    patchfiles-append   patch-ngspice-older-MACH-defines.diff
-
-    depends_lib-append  port:ncurses \
-                        port:libedit \
+	configure.cppflags-prepend -I${prefix}/include/freetype2
+
+    depends_lib-append  port:autoconf \
+    					port:automake \
+    					port:bison \
+                        port:flex \
+                        port:gawk \
+                        port:readline \
+                        port:libtool \
                         port:xorg-libX11 \
                         port:xorg-libXaw \
                         port:xorg-libXext \
```

Remove the dangling backslash.
```suggestion
                        port:xorg-libXext
```